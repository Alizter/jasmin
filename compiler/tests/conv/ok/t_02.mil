fn foo1(x,y : reg u64, c : reg bool) -> reg u64 * reg bool * reg u64 {
  return x,c,y;
}

fn bar1(x,y : reg u64, c : reg bool) -> reg u64 * reg bool * reg u64 {
  return x,c,y;
}

fn test() {
  u, v : reg u64;
  cf : reg bool;
  u_, v_ : reg u64;
  cf_ : reg bool;

  u_,cf_,v_ = foo1(u,v,cf);
  u_,cf_,v_ = bar1(u,v,cf);
  u_,cf_,v_ = bar1(u,v,cf);
  u_,cf_,v_ = foo1(u,v,cf);
}

/*
START:CMD
#ARG="renumber_fun_unique,save[/tmp/before.mil][fun=test],test_conv[test],save[/tmp/after.mil][fun=test]"
ARG="renumber_fun_unique,test_conv[test]"
END:CMD
*/
