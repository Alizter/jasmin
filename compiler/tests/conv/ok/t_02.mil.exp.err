Processed file tests/conv/ok/t_02.mil
