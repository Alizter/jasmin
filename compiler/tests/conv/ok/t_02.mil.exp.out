renumbering identifiers in all function(s)
   REMOVED_TIME
testing conversion in test function(s)
   REMOVED_TIME

total transformation time: REMOVED_TIME

total time (with save/print): REMOVED_TIME
